### ISSUE TITLE
损失异常

### ISSUE THREAD

**Issue by zhoulian-sudo** [beimingli0626/extended-LaPred#1]
作者您好，感谢您出色的研究工作。我在运行你的源代码时遇到了问题，导致大幅度的总损失。请问作者在运行[ByeoungDo Kim]的源代码时候，是否出现clsLoss的出现随着迭代次数逐渐增加。请问问题出在哪里呢？
Hello author, thank you for your excellent research work. I encountered a problem while running your source code, resulting in a significant total loss. May I ask if the author experiences clsLoss when running the source code of [ByteoungDo Kim], as the number of iterations gradually increases. May I ask where the problem lies?
![ClsLoss](https://github.com/beimingli0626/extended-LaPred/assets/71865342/5cb65742-e48f-4185-b497-0b51e5dc62bf)

**Comment by beimingli0626** [comment]
Hi. I haven't actually tested Kim's source code, but I haven't observed such a loss increase at the beginning of training when running my code.

**Comment by zhoulian-sudo** [comment]
> 你好。我还没有实际测试 Kim 的源代码，但是在运行我的代码时，我没有在训练开始时观察到这样的损失增加。

Thank you for your answer. This code has been run many times, but the loss has been consistently abnormal (RegLoss, ClsLoss, allLoss). However, the overall ADE/FDE run is basically consistent with the original paper.
![regloss](https://github.com/beimingli0626/extended-LaPred/assets/71865342/b395c968-7ab5-4ad9-bf9e-f8b890973b92)
![alll_loss](https://github.com/beimingli0626/extended-LaPred/assets/71865342/62a8acb0-52ca-44b1-896a-7a7ecca889f1)

**Comment by beimingli0626** [comment]
Your loss curve makes sense to me since the total training loss consistently decreases throughout the training process. The increase in 'ClsLoss' at the start of training may be because the network is initially learning to reduce the 'RegLoss,' which also appears to consistently decrease.

**Comment by zhoulian-sudo** [comment]
> 你的损失曲线对我来说很有意义，因为在整个训练过程中总训练损失持续减少。训练开始时“ClsLoss”的增加可能是因为网络最初正在学习减少“RegLoss”，而“RegLoss”似乎也在持续减少。

Thank you for your answer. May I ask if you mean that based on these three losses, there are no overall issues with the running model? Or rather, from the perspective of allLoss, there is a slight overfitting, right.
